application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -770,15 +770,15 @@ type Leg {
   For transit legs, intermediate stops between the Place where the leg
   originates and the Place where the leg ends. For non-transit legs, null.
   """
-  intermediatePlaces: [Place]
+  intermediatePlaces: [Place] @deprecated(reason : "Use `leg.stopCalls` instead")
```

Comment:
So this was always just stops despite using the place type?

Reply:
Yes, it was only stops: https://github.com/opentripplanner/OpenTripPlanner/blob/7961877c8a4f33d1e46fefcb498dbce3d456bbc2/application/src/main/java/org/opentripplanner/apis/gtfs/datafetchers/LegImpl.java#L157

 For non-transit legs it returned null.